addons/test/cert-test.c

```diff
@@ -61,3 +63,132 @@ void msc30()
     int rand = 5;
     int a = rand;
 }
+
+void str03()
+{
+    char *string_data=(char*)malloc(16);
+    char a[16];
+    strcpy(string_data,"hello");            //cert-STR07-C
+    strncpy(a, string_data, sizeof(a));     //cert-STR03-C
```

Comment:
so can you add a test case like this also:

    strncpy(a,b,c); d=sizeof(int);